src/components/slug-section/index.tsx

```diff
@@ -63,7 +65,23 @@ export const SlugSection: FC<ISlugSectionProps> = ({ staticPost }) => {
         <div className="min-h-24 w-full">
           <ReleaseNotes post={post} />
         </div>
+        
       </div>
+      <button
+          style={{
```

Comment:
Delete this entire style prop and add the styling through the classname prop instead with TailwindCSS